lib/capistrano-db-tasks/database.rb

```diff
@@ -106,8 +106,13 @@ def dump_cmd_ignore_data_tables_opts
   class Remote < Base
     def initialize(cap_instance)
       super(cap_instance)
-      @config = @cap.capture("cat #{@cap.current_path}/config/database.yml")
-      @config = YAML.load(ERB.new(@config).result)[@cap.fetch(:rails_env).to_s]
+      @cap.within @cap.current_path do
+        @cap.with rails_env: @cap.fetch(:rails_env) do
+          @config = @cap.capture(:rails, 'runner "puts Rails.application.config.database_configuration[Rails.env].to_yaml"', '2>/dev/null')
+          @config.gsub!(/.*\-\-\-/m, '---') # Remove all bundler and rails initialization errors
+        end
```

Comment:
@ktaragorn i don't wanna using rake task, because we can require 'capistrano-*' gems only in development group. On the production (and staging) server they can be unavailable and our rake task too.

Alternative: https://github.com/sgruhier/capistrano-db-tasks/pull/54#issuecomment-125984751 (author @fabn). But save file to `tmp/config.yml` isn't correct. Because tempdir can be one of `[ENV['TMPDIR'], ENV['TMP'], ENV['TEMP'], @@systmpdir, '/tmp', '.']` and can be unequal `tmp/` ([proof](http://ruby-doc.org/stdlib-2.0.0/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir)).